twitter4j-async/pom.xml

```diff
@@ -43,18 +43,9 @@
     </issueManagement>
     <distributionManagement>
         <repository>
-            <id>sonatype-nexus-staging</id>
-            <name>Nexus Release Repository</name>
-            <uniqueVersion>false</uniqueVersion>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/
-            </url>
+          <id>internal</id>
+          <url>http://sprout-archiva01.int.sproutsocial.com:8080/archiva/repository/internal/</url>
```

Comment:
This is broken, right? archiva is dead now.

Reply:
@ccstolley  Yup. you are absolutely right. I got this from updates Aaron made to the pom's (back when archiva did exist).